src/Filter/ScssFilter.php

```diff
@@ -51,4 +54,130 @@ public function input($filename, $input)
         $return = $this->_runCmd($bin, '', array('PATH' => $this->_settings['path']));
         return $return;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDependencies(AssetTarget $target)
+    {
+        $children = [];
+        foreach ($target->files() as $file) {
+            $imports = CssUtils::extractImports($file->contents());
+            if (empty($imports)) {
+                continue;
+            }
+
+            $ext = $this->_settings['ext'];
+            $extLength = strlen($ext);
+
+            $deps = [];
+            foreach ($imports as $name) {
+                if (preg_match('/(.*)\/\*\*\/\*$/', $name, $matches)) {
```

Comment:
I took a look, and I don't think it will be viable to support every sass extension that exists, so I'd rather not add support for sass_glob.